torch_xla_py/xla_model.py

```diff
@@ -369,13 +369,13 @@ def mark_step():
   _mark_step(getattr(_TLS, 'replication', None))
 
 
-def optimizer_step(optimizer, closure=None, barrier=False):
+def optimizer_step(optimizer, barrier=False, optimizer_args={}):
   replication = getattr(_TLS, 'replication', None)
   gradients = _fetch_gradients(optimizer)
   count = len(replication.replication_devices()) if replication else 1
   if count > 1:
     torch_xla._XLAC._xla_cross_replica_sum(gradients, 1.0 / count, [])
-  loss = optimizer.step(closure=closure)
```

Comment:
Is there any way to tell if the `closure` arg is in use anywhere else in the codebase? I checked all the test/test_train_*.py files and they don't use it

Reply:
In our code base we do not use it.
Hard to tell outside, but we are barely out at this point.